### PR TITLE
added  some common header values

### LINKED ISSUE #5: Add autocomplete for some header values
The most obvious one would be if the header key is content-type we can provide an autocomplete list of mime types (and probably just a subset of plain text types + application/json, etc)

### PR DESCRIPTION
**Pull request by jootd**
I tried to replicate what you've done on header keys , and it worked  . 
P.S first time contributing to open source  🫡

Resolves #5

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -143,6 +143,23 @@ func main() {
 		return
 	})
 	cui.RequestHeaderValue.SetLabel("Value: ")
+	cui.RequestHeaderValue.SetAutocompleteFunc(func(currentText string) (entries []string) {
```
Comment (mfinelli) [inline main.go:146]: We only want this autocomplete to run if the header key is `Content-Type`, so at the beginning of this function you should add a guard like so:

```go
if !strings.EqualFold("Content-Type", cui.RequestHeaderKey.GetText()) {
        return
}
```

File: words.go [hunk 2]
```diff
@@ -75,3 +75,16 @@ var commonHeaderKeys = []string{
 	"X-Csrf-Token",
 	"X-Request-ID",
 }
+
+var commonHeaderValues = []string{
```
Comment (mfinelli) [inline words.go]: Could you please name this `commonHeaderContentTypes` or similar? I think we'll probably add other common value autocompletes in the future.

**Review by mfinelli — CHANGES_REQUESTED** [review]
Hi, thanks for contributing! I've left a couple of minor comments. Cheers!

**Comment by mfinelli** [comment]
Resolves #5

File: words.go (hunk 2 above)
Comment (jootd) [reply words.go]: I will change it  asap 🤠

Comment (DataKhukhua) [reply words.go:79]: I will change it  asap 🤠

File: words.go [hunk 3]
```diff
@@ -75,3 +75,16 @@ var commonHeaderKeys = []string{
 	"X-Csrf-Token",
 	"X-Request-ID",
 }
+
+var commonHeaderContentTypes = []string{
```
Comment (mfinelli) [inline words.go:79]: Sorry, one more minor thing. Can you please alphabetize this list and remove `application/zip`? It seems unlikely that someone will be able to enter a proper zip payload into the text area. Thanks

**Review by mfinelli — CHANGES_REQUESTED** [review]

File: words.go (hunk 3 above)
Comment (jootd) [reply words.go:79]: Yes , of course . Thanks for feedback 🫡

**Review by mfinelli — APPROVED** [review]

**Comment by mfinelli** [comment]
This says closed, but I squashed and merged it. Thanks again for contributing!

**Pull request closed without merging**